cmd/abigen/main.go

```diff
@@ -128,6 +130,9 @@ func main() {
 
```

Comment:
Regarding the `bin` here. It should have some additional information like `//$812de29bd0a57d8fddaebd9e2f77569c6c$ -> /solidity/lib.sol:Set`. Let's filter them out first.

Reply:
It only exists in bin file. Never mind.